docs/types/value-types.rst

```diff
@@ -161,18 +161,33 @@ Fixed Point Numbers
     cannot be assigned to or from.
 
 ``fixed`` / ``ufixed``: Signed and unsigned fixed point number of various sizes. Keywords ``ufixedMxN`` and ``fixedMxN``, where ``M`` represents the number of bits taken by
-the type and ``N`` represents how many decimal points are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
+the type and ``N`` represents how many fractional decimal digits are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
 ``ufixed`` and ``fixed`` are aliases for ``ufixed128x18`` and ``fixed128x18``, respectively.
 
+Fixed point types are used to represent fractional numbers with a fixed precision. The number ``1.5`` is stored in a ``fixed8x2`` (8 bit number with
+two fractional digits) as ``1.50``. In the EVM, it is stored in exactly the same way as the number ``150`` would be stored in an ``int8``.
+The largest and smallest numbers representable in a ``fixed8x2`` are ``1.27`` and ``-1.28``, respectively.
+
+Type conversions:
+
+Implicit conversions from integer and fixed point types to ``fixedMxN`` types can be performed,
+as long as all values can still be represented ("widening" conversion).
+
+Explicit conversions to integer types truncate the fractional part (towards zero) and can only be performed if the integer
+type can contain all resulting values after truncation.
+
+Explicit conversions to and from ``bytesNN`` types of matching width can be performed and do not change the internal representation.
+
+
 Operators:
 
 * Comparisons: ``<=``, ``<``, ``==``, ``!=``, ``>=``, ``>`` (evaluate to ``bool``)
 * Arithmetic operators: ``+``, ``-``, unary ``-``, ``*``, ``/``, ``%`` (modulo)
 
 .. note::
     The main difference between floating point (``float`` and ``double`` in many languages, more precisely IEEE 754 numbers) and fixed point numbers is
```

Comment:
Yeah, maybe we should also just drop this paragraph and only explain how fixed point numbers work and how the precision works.

Reply:
I would not drop it completely. I think it should at least say that these are not floating-point numbers and they work differently. From my experience, many people do not know the difference, even experienced devs who were not taught this formally and just never had a need to dig deeper into this topic on their own. Just instead of explaining what the difference is we could link to some external article.